From 44454352e6cf0e6eaa502708ed46fcf664a3732b Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Wed, 2 Mar 2016 10:59:54 -0500
Subject: [PATCH] erlang: remove libatomic_ops

Do not submit this upstream! For all Nerves platforms, this is ok, but
the correct patch should be cleaned up.
---
 package/erlang/Config.in | 1 -
 package/erlang/erlang.mk | 3 ---
 2 files changed, 4 deletions(-)

diff --git a/package/erlang/Config.in b/package/erlang/Config.in
index 5fa2028..c5b21d5 100644
--- a/package/erlang/Config.in
+++ b/package/erlang/Config.in
@@ -8,7 +8,6 @@ config BR2_PACKAGE_ERLANG
 	depends on BR2_USE_MMU # fork()
 	depends on !BR2_STATIC_LIBS
 	depends on BR2_PACKAGE_LIBATOMIC_OPS_ARCH_SUPPORTS
-	select BR2_PACKAGE_LIBATOMIC_OPS
 	help
 	  Erlang is a programming language used to build massively scalable
 	  soft real-time systems with requirements on high availability.
diff --git a/package/erlang/erlang.mk b/package/erlang/erlang.mk
index db6bef9..7d31ec4 100644
--- a/package/erlang/erlang.mk
+++ b/package/erlang/erlang.mk
@@ -30,9 +30,6 @@ ERLANG_CONF_ENV += erl_xcomp_sysroot=$(STAGING_DIR)
 
 ERLANG_CONF_OPTS = --without-javac
 
-ERLANG_DEPENDENCIES += libatomic_ops
-ERLANG_CONF_OPTS += --with-libatomic_ops=$(STAGING_DIR)/usr LIBS=-latomic_ops
-
 # erlang uses openssl for all things crypto. Since the host tools (such as
 # rebar) uses crypto, we need to build host-erlang with support for openssl.
 HOST_ERLANG_DEPENDENCIES = host-openssl
-- 
2.5.0

